integration/v3_lease_test.go

```diff
@@ -438,6 +440,8 @@ func TestV3LeaseFailover(t *testing.T) {
 		t.Fatal(lresp.Error)
 	}
 
+	expectedExp = time.Now().Add(time.Duration(lresp.TTL) * time.Second)
```

Comment:
i investigate a bit more. the reason why proxy doesn't get the first req through is that there is a delay in when lease client sends out ka req to etcd server. https://github.com/coreos/etcd/blob/master/clientv3/lease.go#L456
this delay is long enough for watch detect leader loss before the req being sending out. if i change the delay to 10ms, the behavior is the same as etcd server.